shipyard_blueprints/s3/integration/int.py

```diff
@@ -0,0 +1,245 @@
+import os
```

Comment:
An particular reason to breakaway from pytest naming convention? Typically, test files are located in a `tests` folder and include `test` as a prefix or suffix in their filenames.